blockchain/fundamentals/ethereum-on-the-web/metamask.md

```diff
@@ -0,0 +1,57 @@
+---
+author: mihaiberq
+
+levels:
+
+  - beginner
+
+  - basic
+
+type: normal
+
+category: must-know
+
+standards:
+  blockchain.identify-blockchain-applications.4: 10
+
+aspects:
+
+  - introduction
+
+  - workout
+
+---
+# MetaMask
+
+---
+## Content
+
+Like for bitcoins, you need a wallet to be able to hold any ether. The Ethereum client works well if you are looking to also run a node. In this workout we'll discuss some alternatives.
```

Comment:
what does the wallet have to do with running a node
I found this paragraph really confusing

Reply:
will try to reformulate, but i t refers to the fact that the ethereum client can work as an online wallet.